src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/BOM/MaterialForm/MaterialFormView.tsx

```diff
@@ -117,44 +124,55 @@ const MaterialFormView: React.FC<MaterialFormViewProps> = ({
             />
           </FormControl>
         </Grid>
-        <Grid item xs={4}>
+        <Grid item xs={6}>
           <FormControl fullWidth>
             <FormLabel>Manufacturer</FormLabel>
             <Controller
               name="manufacturerName"
               control={control}
-              defaultValue={control._defaultValues.manufacturerName}
-              render={({ field }) => (
-                <TextField
-                  {...field}
-                  select
-                  variant="outlined"
-                  error={!!errors.manufacturerName}
-                  helperText={errors.manufacturerName?.message}
-                  onChange={(event) => {
-                    const selectedValue = event.target.value;
-                    if (selectedValue === 'createManufacturer') {
-                      const manufacturerName = prompt('Enter Manufacturer Name');
-                      if (manufacturerName) {
-                        createManufacturer(manufacturerName);
-                        field.onChange(event);
-                      }
-                    } else {
-                      field.onChange(event);
-                    }
-                  }}
-                >
-                  {allManufacturers.map((manufacturer) => (
-                    <MenuItem key={manufacturer.name} value={manufacturer.name}>
-                      {manufacturer.name}
-                    </MenuItem>
-                  ))}
-                  <MenuItem value="createManufacturer">+ Create Manufacturer</MenuItem>
-                </TextField>
-              )}
+              render={({ field: { onChange, value } }) => {
+                const mappedManufacturers = allManufacturers
+                  .sort((a, b) => a.name.localeCompare(b.name))
+                  .map(manufacturersToAutocomplete);
+                const onClear = () => {
+                  setValue('manufacturerName', '');
+                  onChange('');
+                };
+                return (
+                  <Box sx={{ alignItems: 'center' }}>
+                    <NERAutocomplete
+                      sx={{ bgcolor: 'inherit' }}
+                      id={'manufacturer'}
+                      size="medium"
+                      options={mappedManufacturers}
+                      value={mappedManufacturers.find((manufacturer) => manufacturer.label === value) || null}
+                      placeholder="Select Manufacturer"
+                      onChange={(_event, newValue) => {
+                        newValue ? onChange(newValue.id) : onClear();
+                      }}
+                    />
+                  </Box>
+                );
+              }}
             />
           </FormControl>
         </Grid>
+        <Grid item xs={6}>
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <NERButton
+              sx={{ width: '100%', height: '55px', marginTop: '23px' }}
```

Comment:
the marginTop should go on the Grid as well, also it looks a bit off center with marginTop: 23px:
<img width="397" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/c7b6c6c7-24b8-4757-ae8a-10ba29d4d9cc">